Library/Formula/berkeley-db.rb

```diff
@@ -33,6 +33,26 @@ def install
       --enable-cxx
       --enable-compat185
     ]
+    if build.with? "java"
+        java_home = ""
+        if ENV.has_key?("JAVA_HOME")
+            java_home = ENV["JAVA_HOME"]
+       else
+           java_home = `/usr/libexec/java_home`.chomp
```

Comment:
Ok, thats annoying because bdb 5.3.28 won't build with jdk 1.8 which is why I switched to using JDK 1.7 :)

env CLASSPATH="./classes" java -classpath ./classes \
	    com.sleepycat.persist.model.ClassEnhancer ./classes
java.lang.IllegalArgumentException
	at com.sleepycat.asm.ClassReader.<init>(ClassReader.java:167)
	at com.sleepycat.asm.ClassReader.<init>(ClassReader.java:153)
	at com.sleepycat.persist.model.ClassEnhancer.enhanceBytes(ClassEnhancer.java:297)
	at com.sleepycat.persist.model.ClassEnhancer.enhanceFile(ClassEnhancer.java:246)
	at com.sleepycat.persist.model.ClassEnhancer.enhanceFile(ClassEnhancer.java:242)
	at com.sleepycat.persist.model.ClassEnhancer.enhanceFile(ClassEnhancer.java:242)
	at com.sleepycat.persist.model.ClassEnhancer.enhanceFile(ClassEnhancer.java:242)
	at com.sleepycat.persist.model.ClassEnhancer.enhanceFile(ClassEnhancer.java:242)
	at com.sleepycat.persist.model.ClassEnhancer.main(ClassEnhancer.java:108)
Exception in thread "main" java.lang.IllegalArgumentException
	at com.sleepycat.asm.ClassReader.<init>(ClassReader.java:167)
	at com.sleepycat.asm.ClassReader.<init>(ClassReader.java:153)
	at com.sleepycat.persist.model.ClassEnhancer.enhanceBytes(ClassEnhancer.java:297)
	at com.sleepycat.persist.model.ClassEnhancer.enhanceFile(ClassEnhancer.java:246)
	at com.sleepycat.persist.model.ClassEnhancer.enhanceFile(ClassEnhancer.java:242)
	at com.sleepycat.persist.model.ClassEnhancer.enhanceFile(ClassEnhancer.java:242)
	at com.sleepycat.persist.model.ClassEnhancer.enhanceFile(ClassEnhancer.java:242)
	at com.sleepycat.persist.model.ClassEnhancer.enhanceFile(ClassEnhancer.java:242)
	at com.sleepycat.persist.model.ClassEnhancer.main(ClassEnhancer.java:108)
make: *** [db.jar] Error 1

I'll check to see if there is something that I can do here.